rpcs3/rpcs3qt/game_compatibility.h

```diff
@@ -4,9 +4,10 @@
 
 #include <QPainter>
 #include <QJsonObject>
-#include <QNetworkAccessManager>
-#include <QNetworkRequest>
-#include <QTimer>
+
+#define NOMINMAX
+#define CURL_STATICLIB
+#include <curl/curl.h>
```

Comment:
CURL probably can be declared as a forward declaration under extern "C"{}, with an actual include in cpp file.